draft-kuhn-quic-bdpframe-extension-01.xml

```diff
@@ -405,21 +424,17 @@ BDP Frame {
 
 		<t>
                 The server SHOULD NOT trust
-                the content of the BDP Frame received from the client. 
+                the content of the BDP Frame received from the client if the secured hash is not computed by the sender. 
 		Even if the QUIC 
                 packets containing the BDP Frame are
                 encrypted, a client could modify the
                 values within the extension and
                 encrypt the QUIC packet.
-                Authentication mechanisms
-                might not guarantee that
-                the values are safe.
-                It is not an easy operation for a client
-                to modify authenticated or encrypted data
-                without this being detected by a server.
-                Modification could be realized by malicious clients.
                 One way to avoid this is for a server to also 
                 store the saved_rtt and saved_bb parameters.
+		Another way to avoid this is to exploit the secured hash generated by the sender. If the client 
+		modify a transport parameter, the result of the hash would be different. The sender should then 
```

Comment:
/exploit/use/ ... to avoid the security people reading this as an exploitation. (and see also below for the same word)

Reply:
Thanks! Fixed. https://github.com/NicoKos/resume_connection/commit/b6cf8df1573405b060e3cfcb62b1e0efcb38146d